src/test/java/hudson/plugins/sshslaves/agents/AgentConnectionBase.java

```diff
@@ -46,11 +46,15 @@ public class AgentConnectionBase {
   @ClassRule
   public static CheckIsDockerAvailable isDockerAvailable = new CheckIsDockerAvailable();
 
-  @Rule
+  @Rule(order = 10)
```

Comment:
Previously, the 3m default timeout in `JenkinsRule` superseded the expressed timeout here.